comfy_cli/auth/store.py

```
@@ -0,0 +1,350 @@
+    if not isinstance(body, dict):
+        return None
+    key = body.get("key")
+    if not isinstance(key, str) or not key:
+        return None
+    return AuthRecord(provider=provider, key=key, updated_at=str(body.get("updated_at", "")))
+
+
+_SAFE_PROVIDER = re.compile(r"^[a-zA-Z0-9_\-]{1,64}$")
+
+
+def set(provider: str, key: str) -> AuthRecord:
```

Pylint (4.0.5)

[warning] Redefining built-in 'set'

(W0622)